### PR TITLE
fix(async-setup): update faq link to new docs url

### PR DESCRIPTION
**Pull request by MaskDuck**
## Checklist

- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] If code changes were made then they have been tested. (no need)
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by TAG-Epic — APPROVED** [review]

File: nextcord/ext/commands/bot.py [hunk 1]
```diff
@@ -701,7 +701,7 @@ def _load_from_module_spec(
                     raise RuntimeError(f"""
                     Looks like you are attempting to load an asynchronous setup function incorrectly.
                     Please read our FAQ here:
-                    https://nextcord.readthedocs.io/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
+                    https://docs.nextcord.dev/en/latest/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
```
Comment (TAG-Epic) [inline nextcord/ext/commands/bot.py:704]:
```suggestion
                    https://docs.nextcord.dev/en/stable/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it
```

Comment (ooliver1) [reply nextcord/ext/commands/bot.py:704]: the stable branch/link is a bit outdated though

**Comment by MaskDuck** [comment]
@TAG-Epic pushed upgrade

**Review by ooliver1 — APPROVED** [review]
not sure what the [breaking] means

**Comment by MaskDuck** [comment]
mm it display a breaking change

**Comment by ooliver1** [comment]
im unsure as to how changing an error msg is breaking

**Pull request merged**